### PR TITLE
Hex spec support and commons utils

### PR DESCRIPTION
**Pull request by tangrammer**
# Description

Complete wallet-address spec with generator support to be able to generate mock random valid wallets 

About generation in clj vs cljs also relates to https://github.com/kubelt/kubelt/pull/335

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

```clojure
(require '[malli.generator :as mg] '[malli.core :as m])
(m/validate wallet-address "0x3E2C108FEE24bC552Ba98e3360A97d0912Cc0D63")
;; => true
(mg/generate com.kubelt.gen.wallet/wallet-address)
;; => "0xE7187321fdb2A8E78aca4122586280571fa21D88"

```
## New unit tests
  - [com.kubelt.spec.common-test](https://github.com/kubelt/kubelt/blob/wallet-address-hex-spec/src/test/com/kubelt/spec/common_test.cljc)
  - [com.kubelt.spec.wallet-test](https://github.com/kubelt/kubelt/blob/wallet-address-hex-spec/src/test/com/kubelt/spec/wallet_test.cljc)

### CODE REVIEW

File: src/main/com/kubelt/spec/crypto.cljc [hunk 1]
```diff
@@ -23,9 +26,9 @@
    [:digest/byte-length spec.common/byte-length]
    [:digest/bit-length spec.common/bit-length]
    [:digest/bytes spec.common/byte-data]
-   [:digest/hex-string hex-string]])
+   [:digest/hex-string hex-64-string]])
 
-(def digest-spec
+#_(def digest-spec
```
Comment (tangrammer) [inline src/main/com/kubelt/spec/crypto.cljc:31]: commenting this not-in-use definition

**Review by drew-patel — DISMISSED** [review]

**Comment by tangrammer** [comment]
@crimeminister changes  added to decouple generation details, could you review them?

**Comment by tangrammer** [comment]
Example of connecting pieces 
https://github.com/kubelt/three-id/pull/21

**Comment by crimeminister** [comment]
@tangrammer One more bit of re-org: could you please move the `.gen` namespace under `.lib`? Current pattern is that namespaces directly under `com.kubelt` are (more or less) compilation targets or eventual standalone libraries.

**Review by crimeminister — APPROVED** [review]

**Pull request merged**